os/drivers/audio/alc5658.c

```diff
@@ -781,7 +830,7 @@ static int alc5658_shutdown(FAR struct audio_lowerhalf_s *dev)
 
 	alc5658_takesem(&priv->devsem);
 	sq_entry_t *tmp = NULL;
-	for (tmp = (sq_entry_t *)sq_peek(&priv->pendq); tmp; tmp = sq_next(tmp)) {
+	for (tmp = (sq_entry_t *) sq_peek(&priv->pendq); tmp; tmp = sq_next(tmp)) {
```

Comment:
space.

Reply:
This is modified by our formatter script.